client/signup/main.jsx

```diff
@@ -599,6 +599,8 @@ class Signup extends React.Component {
 	}
 
 	getFlowLength() {
+		// Since no action is required in the final step, we exclude it from the counter.
+		flows.excludeStep( 'launch' );
```

Comment:
I believe this imperative approach won't exclude the step only from the counter so it may have some unexpected side effects. I've suggested a declarative way to achieve the same effect in a separate PR: https://github.com/Automattic/wp-calypso/pull/54318
Also, we can merge that one (after review) without waiting for strings to be fully translated.

Reply:
Yup! I prefer that one for sure. This call has long-lasting side effects.